instrumentation/restlet/restlet-1.0/javaagent/build.gradle.kts

```
@@ -0,0 +1,26 @@
+plugins {
+  id("otel.javaagent-instrumentation")
+}
+
+muzzle {
+  pass {
+    group.set("org.restlet")
+    module.set("org.restlet")
+    versions.set("[1.0.0, 1.2-M1)")
```

How about adding
```suggestion
    versions.set("[1.0.0, 1.2-M1)")
    assertInverse.set(true)
```
to verify that other versions do not match?